### PR TITLE
Merge reading lists feature branch

### PR DESCRIPTION
**Pull request by tgr**
Bug: [T181107](https://phabricator.wikimedia.org/T181107)

### CODE REVIEW

**Comment by d00rman** [comment]
@tgr the PR needs a rebase against current master.

**Comment by Pchelolo** [comment]
@tgr merge conflicts...

File: projects/dev.yaml [hunk 1]
```diff
@@ -37,6 +37,8 @@ paths:
               x-modules:
                 - path: v1/citoid.js
                   options: '{{options.citoid}}'
+                - path: v1/lists.js
+                  options: '{{options.lists}}'
```
Comment (Pchelolo) [inline projects/dev.yaml]: Just to verify I'm correct - the options are not actually ever used, right? I'm asking cause I don't see them being specified in the `config.wikimedia.yaml` so I guess this was added for the future uses and we don't need to accompany this PR with a prod config change.

**Comment by Pchelolo** [comment]
I've made a final pass through the code and everything looks good. After a rebase this is good to go 🎉

File: projects/dev.yaml (hunk 1 above)
Comment (tgr) [reply projects/dev.yaml]: Yeah, it's not used currently. I will probably use it for [T180416](https://phabricator.wikimedia.org/T180416) but nothing substantial.

**Comment by tgr** [comment]
Rebased. I don't think the test error is related.

I added one more trivial patch (#926).

**Pull request merged**